core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -218,19 +218,29 @@
   WORKER_TIERED_STORE_LEVEL0_DIRS_PATH(Name.WORKER_TIERED_STORE_LEVEL0_DIRS_PATH, "/mnt/ramdisk"),
   WORKER_TIERED_STORE_LEVEL0_DIRS_QUOTA(Name.WORKER_TIERED_STORE_LEVEL0_DIRS_QUOTA,
       "${alluxio.worker.memory.size}"),
-  WORKER_TIERED_STORE_LEVEL0_RESERVED_RATIO(Name.WORKER_TIERED_STORE_LEVEL0_RESERVED_RATIO, "0.1"),
```

Comment:
can we still keep this but mark it deprecated? we typically only remove properties after major release.